src/scripts/main.js

```diff
@@ -1,7 +1,46 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.querySelector('body');
+
+  const card = document.createElement('div');
+
+  card.style.top = `${posTop}px`;
+  card.style.right = `${posRight}px`;
+  card.classList.add('notification');
+
+  if (type === 'success') {
+    card.classList.add('success');
```

Comment:
Ты type передаешь просто в класс, зачем тут эти три if?
```suggestion
  card.classList.add(type);
```